streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -186,13 +195,25 @@ public void testStorePartitions() throws IOException {
     @Test
     public void testUpdateNonInitializedStore() throws IOException {
         final StreamsConfig config = createConfig(baseDir);
-        task = new StandbyTask(taskId, topicPartitions, topology, consumer, changelogReader, config, null, stateDirectory);
+        task = new StandbyTask(
+            taskId, topicPartitions, topology, consumer, changelogReader, config, streamsMetrics, stateDirectory);
```

Comment:
Nit: We try to avoid 'intermediate' line breaks in parameter lists: it should either fit all on one line `new StandbyTask(a, b, c, d);`, or each parameter should have its own line.

There's less consistency about how to align the arg list when splitting lines. Both of these are fine:
```java
new StandbyTask(a,
                b,
                c);
```
and
```java
new StandbyTask(
    a,
    b,
    c
);
```

Feel free to apply your preference. Please also apply this elsewhere.

Reply:
Done